### ISSUE TITLE
[Topic] Readability

### ISSUE THREAD

**Issue by rianrietveld** [wpaccessibility/wp-a11y-docs#120]
URL: https://wpaccessibility.org/docs/topics/content/readability/
Code: https://github.com/wpaccessibility/wp-a11y-docs/blob/main/docs/topics/content/readability.md

Please read [How to contribute](https://wpaccessibility.org/docs/contribute/) before you start.

## Tasks

- This content should be reviewed, restructured and expanded.
- Make clear the readability is not required for WCAG AA but is considered best practice
